app/assets/javascripts/annotations.js

```diff
@@ -294,10 +295,13 @@ function displayAnnotations() {
 
   _.each(annotationsByPositionByLine[currentHeaderPos], function (arr_annotations, line) {
     _.each(arr_annotations, function (annotationObj, ind) {
+      if (annotationObj.global_comment === true)
```

Comment:
Do we need an equality check here? would `annotationObj.global_comment` by itself work?

Reply:
Good point, that would work